packages/soy-contracts/truffle-config.js

```diff
@@ -8,19 +29,36 @@ module.exports = {
     coverage: {
       host: 'localhost',
       network_id: '*',
-      port: 8555, // <-- If you change this, also set the port option in .solcover.js.
-      gas: 0xfffffffffff, // <-- Use this high gas value
-      gasPrice: 0x01 // <-- Use this low gas price
+      port: 8555,
+      // Values set from docs https://github.com/sc-forks/solidity-coverage#network-configuration
+      gas: 0xfffffffffff,
+      gasPrice: 0x01
+    },
+    mainnet: {
+      provider: provider('mainnet'),
+      network_id: '1',
+      // When deploying to mainnet check https://ethgasstation.info/ for
+      // current gas price. "Standard" suggested
+      gasPrice: Web3.utils.toWei('0', 'gwei')
```

Comment:
Trying to prevent someone from spending gas without thinking about the price.

I was looking for an api to ethgasstation as metamask is clearly using something like it. I didn't sink a lot of time and didn't find anything. Manually checking for now seems fine to me.

Reply:
hmm... strange that it's hard to find that API. Seems pretty useful.

Reply:
I found it! https://github.com/MetaMask/metamask-extension/blob/7c60414075877b79b3b98be82cdb6eda5df10329/ui/app/ducks/gas.duck.js#L187